nc-games/src/components/Reviews.jsx

```diff
@@ -1,34 +1,82 @@
 import { useEffect } from "react";
 import { useState } from "react";
-import { useParams } from "react-router-dom";
+import {
+  useParams,
+  useSearchParams,
+  createSearchParams,
+} from "react-router-dom";
 import { fetchReviews } from "../api";
 import ListedReview from "./ListedReview";
 
 const Reviews = () => {
   const [reviews, setReviews] = useState();
   const [isLoading, setIsLoading] = useState(true);
+  const [sorter, setSorter] = useState("created_at");
+  const [order, setOrder] = useState("desc");
+  const [searchQueries, setSearchQueries] = useSearchParams();
 
   const { category } = useParams();
 
   useEffect(() => {
     setIsLoading(true);
```

Comment:
we don't necessarily need to set isLoading to true here as it's already set to true on line 13